### PR TITLE
[type:feature] add basic-auth

### PR DESCRIPTION
**Pull request by Romic77**
Make sure that:

- [x] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor-guide).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `./mvnw clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: shenyu-admin/src/main/resources/application.yml [hunk 1]
```diff
@@ -66,10 +66,10 @@ shenyu:
 #          namespace:
 #          accessKey:
 #          secretKey:
-#    etcd:
-#      url: http://localhost:2379
-#    consul:
-#      url: http://localhost:8500
+    etcd:
```
Comment (yu199195) [inline shenyu-admin/src/main/resources/application.yml:69]: what  is this？

File: shenyu-admin/src/main/java/org/apache/shenyu/admin/controller/PlatformController.java [hunk 2]
```diff
@@ -63,6 +65,30 @@ public ShenyuAdminResult loginDashboardUser(final String userName, final String
...
     }
 
+    /**
+     * basic auth login.
+     *
+     * @param authorization authorization
+     * @return {@linkplain ShenyuAdminResult}
+     */
+    @GetMapping("/basicAuth")
+    public ShenyuAdminResult httpBasicAuth(@RequestHeader("Authorization") final String authorization) {
```
Comment (yu199195) [inline shenyu-admin/src/main/java/org/apache/shenyu/admin/controller/PlatformController.java:75]: what is this？

File: shenyu-admin/src/main/resources/application.yml (hunk 1 above)
Comment (Romic77) [reply shenyu-admin/src/main/resources/application.yml]: sorry

File: shenyu-admin/src/main/java/org/apache/shenyu/admin/controller/PlatformController.java (hunk 2 above)
Comment (Romic77) [reply shenyu-admin/src/main/java/org/apache/shenyu/admin/controller/PlatformController.java:75]: It's basic auth, what's wrong?

**Pull request closed without merging**